### PR TITLE
Budget App: Project submission

### PR DESCRIPTION
**Pull request by forLoop94**
### The following changes were implemented:

**Splash screen**

A simple page with the name of your app (yes, you need to choose one), and links to the sign up and log in pages.
Sign up and log in pages

The user should be able to register in the app with full name, email and password (all mandatory).
The user can log into the app using email and password.
If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).

**Home page (categories page)**

    When the user logs in, they are presented with the categories page.
    For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
    When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
    There is a button "add a new category" at the bottom that brings the user to the page to create a new category.

**Transactions page**

    For a given category, the list of transactions is presented, ordered by the most recent.
    At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
    There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
    When the user clicks on the "Back" button (<), the user navigates to the home page.

**"Add a new category" page**

    The user fills out a form to create a new category, indicating their name and icon (both mandatory).
    The user clicks (or taps) the "Save" button to create the new category, and is taken to the home page on success.
    When the user clicks on the "Back" button (<), the user navigates to the home page.

**"Add a new transaction" page**

    The user fills out a form to create a new transaction with:
    name (mandatory)
    amount (mandatory)
    categories (mandatory at least one)
    The user click (or taps) the "Save" button to create the new transaction, and is taken to the transactions page for that category.
    When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.

**Testing requirements**
  Create unit and integration tests for all the most important components of your RoR application.

**Technical requirements**
    You should use Postgres as your database.
    You should use devise for authentication.
    You should validate all user input to make sure that anyone with bad intentions cannot compromise your app.
    You can use a view template engine of your choice (.erb, .slim, .haml).
    The project should be deployed and accessible online.
    Your database schema should reflect the following structure:

**Project documentation**
Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

a description of the project.
a demo of the project features.
you should also highlight some interesting piece of code or something you built that you are very proud of.

Presentation video: [Video](https://www.loom.com/share/36ae6cb1537b428596be98fbb1ab9c7b)

### CODE REVIEW

File: app/assets/stylesheets/welcome.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+.topicm {
+  font-size: 3rem;
+  font-weight: 800;
+  margin-bottom: 10rem;
+}
+
+.sign-up {
+  background-color: transparent !important;
+  color: black;
+}
```
Comment (binhussen) [inline app/assets/stylesheets/welcome.css:29]: - Kindly note that your design is not sticked the given design please follow `Colors`
, `Typography` and `Layout` of the given design for **all your pages** to fit the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md) 👍 



| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

🤝️

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @forLoop94 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Built Budget Track App✔️
- Well Structured Code ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/assets/stylesheets/welcome.css (hunk 1 above)
Comment (forLoop94) [reply app/assets/stylesheets/welcome.css:29]: Thank you @binhussen,  for your recommendations. I have made the page displaying the categories have the same styling as the page above.

**Review by IngDavidTM — APPROVED** [review]
Hi @forLoop94 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**